llvm/test/CodeGen/WebAssembly/simd-bitselect.ll

```diff
@@ -0,0 +1,71 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 5
+; RUN: llc < %s  -O3 -verify-machineinstrs -disable-wasm-fallthrough-return-opt -wasm-disable-explicit-locals -wasm-keep-registers -mattr=+simd128 | FileCheck %s
+target triple = "wasm32-unknown-unknown"
+
+define <4 x i32> @bitselect_splat_first_zero_and_icmp(<4 x i32>  %input) {
```

Comment:
Can we move these tests into simd-select.ll to keep it with the other select tests? And reuse the same variable names in that test e.g. use %x for the input, %c for the icmp and %res for the select